Correct function def of gray_to_bin and bin_to_gray

Malkhan52 opened the pull request:
Hello everyone!
This is my first commit to sympy.
I corrected definition of two functions named **bin_to_gray** and **gray_to_bin**.
There is an issue [#117](https://github.com/sympy/sympy-live/issues/117) in sympy-live repo but I checked same was because of sympy, so I send PR here.
I have tested with some examples.

Malkhan52 committed:
Correct function def of gray_to_bin and bin_to_gray

smichr commented on sympy/combinatorics/graycode.py:
```diff
@@ -329,11 +329,17 @@ def gray_to_bin(bin_list):
     >>> gray_to_bin('100')
     '111'
 
+    example taken from issue #117 from:- https://github.com/sympy/sympy-live/issues/117
```
The `test_graycode` file would be a good place for this.

```python
def test_live_issue_117():
    assert bin_to_gray('0100') == '0110'
    assert bin_to_gray('0101') == '0111'
    for bits in ('0100', '0101'):
        assert gray_to_bin(bin_to_gray(bits)) == bits
```

These were confirmed with [online calculator](https://ncalculators.com/digital-computation/binary-gray-code-converter.htm).

I also stopped the tests since you spelled `gray` as `gtay` in the doctest.

Malkhan52 replied:
Thanks for reviewing.
& sorry for that typo.

asmeurer replied:
You don't need to put a reference to the issue in the docstring. Putting it in the commit message is sufficient for provenance.

Malkhan52 committed:
Add a test in test_graycode.py

smichr committed:
remove docstring examples

Examples are included in the test suite.

smichr committed:
Update test_graycode.py

Add newline.

Note: doctests taken from issue at https://github.com/sympy/sympy-live/issues/117

smichr commented:
There were some PEP8 issues that I corrected: a line with space and a file with no newline at the end. I also removed the docstring tests that were essentially duplicated in the test suite.

smichr committed:
import gray_to_bin
